source/userguide/kb.rst

```diff
@@ -668,3 +697,7 @@ In the example below, we search for restaurants whose names best match ``firetra
 		},
 	  	...
 	  ]
+
+.. note::
+
+   We can use the ``size`` field to return a variable number of records.
```

Comment:
Why are we adding the same sentence in two places?

Reply:
Since the actual syntax is a little bit different between query, filter, and sort, I just want to make sure that the user knows they can use the same field `size`.